packages/odyssey-react/src/components/Table/TableHeaderCell.tsx

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import type { ReactNode, DetailedHTMLProps, ThHTMLAttributes } from 'react';
+import { CellTextFormats } from './Table';
```

Comment:
minor but consider the explicit `import type { CellTextFormats } from './Table'` syntax when using composed type annotations or interfaces like this!

Reply:
👍